### PR TITLE
[doc][yba] Clarify smart resize

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify smart resize

@netlify /preview/yugabyte-platform/manage-deployments/edit-universe/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 1]
```diff
@@ -24,39 +24,50 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
...
 
 To change the number of nodes of universes created with an on-premises cloud provider and secured with third-party certificates obtained from external certification authorities, you must first add the certificates to the nodes you will add to the universe. Refer to [Add certificates](../../security/enable-encryption-in-transit/add-certificate-ca/). Ensure that the certificates are signed by the same external CA and have the same root certificate. In addition, ensure that you copy the certificates to the same locations that you originally used when creating the universe.
 
-### Smart resize
+## Smart resize
+
+Normally when resizing a universe, YugabyteDB moves the data from the old nodes to the new nodes. However, if the universe is deployed on AWS, GCP, or Azure using a [cloud provider configuration](../../configure-yugabyte-platform/aws/), you can change the instance type and increase the volume size without migrating the data. This is referred to as smart resize, and can be significantly faster than a full copy of the data.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:53]: minor: maybe we could just say "you can perform certain resizing operations without migrating" given we already list them more precisely below.

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 2]
```diff
@@ -24,39 +24,50 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
...
+
+Smart resize is available for the following operations:
+
+- Change the Instance type.
 
-For universes that use Google Cloud Provider (GCP), Amazon Web Services (AWS), or Microsoft Azure, YugabyteDB Anywhere allows you to change the VM images and increase the volume size without moving the data from the old nodes to the new nodes. This is known as smart resize and is subject to the following:
+    Note that smart resize is not available when changing the instance type from an AWS EBS-backed instance type (like c5.xlarge) to a local storage-backed instance type (like i3.xlarge), or vice-versa.
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:59]: minor: local-storage-backed? Not sure about hyphenation convention in English :)

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 3]
```diff
@@ -24,39 +24,50 @@ YugabyteDB Anywhere allows you to expand a universe to add more nodes and shrink
...
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Regions** - you can select any region configured in the provider used to deploy the universe.
+    - [Master Placement](../../create-deployments/dedicated-master/).
+    - **Total Nodes** and **Availability Zones** - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
+    - [User tags](../instance-tags/).
```
Comment (iSignal) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:41]: Technically, changing just the user tags for a cloud provider also is in-place (doesn't require touching the YBDB processes as it is a cloud provider option). Don't think we want to call it a "smart resize" but maybe we could say something like 

- User tags (changing just the tags for a cloud provider based universe does not involve any node restarts or data moves)

But we could also ignore this case.

**Review by iSignal — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 4]
```diff
@@ -14,49 +14,61 @@ menu:
...
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Cloud Configuration**
+        - **Regions** - Select any region configured in the provider used to deploy the universe.
+        - [Master Placement](../../create-deployments/dedicated-master/).
+        - **Total Nodes** and **Availability Zones** - As you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```
Comment (rajmaddy89) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:41]: Under Instance Configuration, 
1. we can also resize the 'Storage Type' and `Volume Count` of the Instance, but that amount to a Full move

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 5]
```diff
@@ -14,49 +14,61 @@ menu:
...
+    - [User Tags](../instance-tags/). Changing tags doesn't require any node restarts or data migration.
+
+    Note that you can't change the replication factor of a universe.
```
Comment (rajmaddy89) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:44]: May we can include
- **Advanced Configuration** - change communication ports such as masterHttpPort, masterRpcPort, tserverHttpPort, tserverRpcPort. Editing any of these ports causes a Full Move

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 6]
```diff
@@ -14,49 +14,61 @@ menu:
...
+- Change the Instance type.
 
-For universes that use Google Cloud Provider (GCP), Amazon Web Services (AWS), or Microsoft Azure, YugabyteDB Anywhere allows you to change the VM images and increase the volume size without moving the data from the old nodes to the new nodes. This is known as smart resize and is subject to the following:
+    Note that smart resize is not available when changing the instance type from an AWS EBS-backed instance type (like c5.xlarge) to a local storage-backed instance type (like i3.xlarge), or vice-versa.
 
-- For Azure universes, you can't increase the volume size for ultra SSDs.
+- Increase the Volume disk size.
 
-- To avoid potential data loss, you can't do a smart resize of instances with ephemeral disks. Smart resize _to_ ephemeral disks is supported.
+    Note that smart resize is not available with Azure ultra disks, or when decreasing the size.
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:64]:
```suggestion
    Note that smart resize is not available with Azure ultra disks, or when decreasing the volume size.
```

**Review by aishwarya24 — COMMENTED** [review]
left a comment

**Comment by aishwarya24** [comment]
LGTM

**Review by aishwarya24 — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md [hunk 7]
```diff
@@ -14,49 +14,64 @@ menu:
...
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Cloud Configuration**
+        - **Regions** - Select any region configured in the provider used to deploy the universe.
+        - [Master Placement](../../create-deployments/dedicated-master/).
+        - **Total Nodes** and **Availability Zones** - As you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md:41]:
```suggestion
    - **Instance Configuration** - Change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```

File: docs/content/stable/yugabyte-platform/manage-deployments/edit-universe.md [hunk 8]
```diff
@@ -12,49 +12,64 @@ menu:
...
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Cloud Configuration**
+        - **Regions** - Select any region configured in the provider used to deploy the universe.
+        - [Master Placement](../../create-deployments/dedicated-master/).
+        - **Total Nodes** and **Availability Zones** - As you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/manage-deployments/edit-universe.md:39]:
```suggestion
    - **Instance Configuration** - Change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```

File: docs/content/v2.20/yugabyte-platform/manage-deployments/edit-universe.md [hunk 9]
```diff
@@ -12,49 +12,64 @@ menu:
...
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/concepts/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Cloud Configuration**
+        - **Regions** - Select any region configured in the provider used to deploy the universe.
+        - [Master Placement](../../create-deployments/dedicated-master/).
+        - **Total Nodes** and **Availability Zones** - As you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```
Comment (ddhodge) [inline docs/content/v2.20/yugabyte-platform/manage-deployments/edit-universe.md:39]:
```suggestion
    - **Instance Configuration** - Change instance type and storage volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
```

**Pull request merged**